### PR TITLE
Tayfun week 1

### PR DESCRIPTION
**Pull request by tayfuntayfun**

### CODE REVIEW

File: week-1/Homework/mandatory/3-function-output.js [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
 // Add comments to explain what this function does. You're meant to use Google!
+// This function concatenates the second string after the first string as seen below.
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+
+    return firstWord.concat(` ${secondWord} ${thirdWord}`);
```
Comment (brightpixels) [inline week-1/Homework/mandatory/3-function-output.js]: This works but there is a better way to do this.

File: week-1/Homework/mandatory/4-tax.js [hunk 2]
```diff
@@ -5,7 +5,14 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+
```
Comment (brightpixels) [inline week-1/Homework/mandatory/4-tax.js]: Well done. This works but can you think of anything else to make break down this code so it's more readable. Imagine your work colleague doesn't know what 0.20 represents. What can you do to make code readable.

File: week-1/Homework/mandatory/4-tax.js [hunk 3]
```diff
@@ -17,7 +24,14 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function formatCurrency() {}
+function formatCurrency(productPrice) {
```
Comment (brightpixels) [inline week-1/Homework/mandatory/4-tax.js]: - I like the way you make `£` a variable on its own first.
- The way you named the variable `SalesPrice` doesn't follow convention. Can you work out what you are doing wrong here?
- `formatCurrency` tests are failing. That means your function implementation is wrong. You are close but missing important functionality that was requested in the comments: `prices must have 2 decimal places`

File: week-1/Homework/mandatory/3-function-output.js (hunk 1 above)
Comment (tayfuntayfun) [reply week-1/Homework/mandatory/3-function-output.js]: Hi Abdo, here is a different way of scripting this part.

File: week-1/Homework/mandatory/3-function-output.js [hunk 4]
```diff
@@ -1,14 +1,20 @@
...
+
+    //return firstWord.concat(" " + secondWord + " " + thirdWord);
     // Write the body of this function to concatenate three words together
     // Look at the test case below to understand what to expect in return 
 }
```
Comment (tayfuntayfun) [inline week-1/Homework/mandatory/3-function-output.js]: Changed how concat gets the strings from the function.

File: week-1/Homework/mandatory/4-tax.js (hunk 3 above)
Comment (tayfuntayfun) [reply week-1/Homework/mandatory/4-tax.js]: camelCase :), you are right. I have also added .toFixed function to have 2 digit decimal added to the product price. Hope I have done the right thing.

**Pull request closed without merging**